.github/workflows/CICD.yaml

```
@@ -43,14 +43,18 @@ jobs:
           cat package.json
           rm -rf node_modules package-lock.json
           npm install
-          npm run publish
+          npm run package
```

```suggestion
          npm run package
          npm run publish --version ${{ needs.semantic.outputs.version }} --files release/${{ needs.semantic.outputs.version }}/*
```
Simplify your steps, do it all here